packages/ros1/src/SystemHandle.cpp

```diff
@@ -175,21 +176,25 @@ std::shared_ptr<TopicPublisher> SystemHandle::advertise(
     const std::string& message_type,
     const YAML::Node& configuration)
 {
+  const YAML::Node& queue_config = configuration["queue_size"];
+  const YAML::Node& latch_config = configuration["latch"];
+
+  int queue_size = queue_config ? queue_config.as<int>() : default_queue_size;
+  bool latch_behavior = latch_config ? latch_config.as<bool>() : default_latch_behavior;
```

Comment:
Same as before, we can reduce these four lines to two by just passing the default value to the `as<T>(~)` function.